AAUnlimited/Texts/mod/poser/posemgr.lua

```diff
@@ -124,7 +124,10 @@ function deleteposebutton.action()
 			log.error(msg)
 		end
 		log.spam("Removed %s", path)
-		populateposelist()
+		poselist.valuestring = posename.value
+		if poselist.valuestring == posename.value then
```

Comment:
no
*writting* to `valuestring` changes the selected item if the assigned value is found
otherwise it remains unchanged
thus, *reading* from `valuestring` at the end will be different if the value doesn't exist